CMakeLists.txt

```diff
@@ -790,6 +790,7 @@ install(
   FILES "${CMAKE_CURRENT_BINARY_DIR}/CHANGELOG.md"
         "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE"
         "${CMAKE_CURRENT_SOURCE_DIR}/VAST.spdx"
+        "${CMAKE_CURRENT_SOURCE_DIR}/python/vast-python.spdx"
```

Comment:
How about we call these files `sbom.spdx` and `sbom-python.spdx`? Perhaps with a `vast-` prefix to make it even clearer.

Reply:
I just appended the entries to the existing `VAST.spdx` for now to minimize the amount of changes.